core-api/src/main/java/com/optimizely/ab/Optimizely.java

```diff
@@ -160,8 +161,8 @@ private Optimizely(@Nonnull ProjectConfig projectConfig,
         LogEvent impressionEvent =
                 eventBuilder.createImpressionEvent(projectConfig, experiment, variation, userId, attributes);
         logger.info("Activating user \"{}\" in experiment \"{}\".", userId, experiment.getKey());
-        logger.debug("Dispatching impression event to URL {} with params {}.", impressionEvent.getEndpointUrl(),
-                     impressionEvent.getRequestParams());
+        logger.debug("Dispatching impression event to URL {} with params {} and payload \"{}\".",
+                     impressionEvent.getEndpointUrl(), impressionEvent.getRequestParams(), impressionEvent.getBody());
```

Comment:
How expensive would this check really be?  It's just a constant time operation to check whether or not `eventBuilder instanceof EventBuidlerV2` right?  Printing out an empty `{}` seems like it would be confusing to someone reading the logs.